### PR TITLE
no defaults for vX reference tickers

### PR DESCRIPTION
**Pull request by jbonzo**

### CODE REVIEW

**Review by qrpike — APPROVED** [review]

**Review by mmoghaddam385 — APPROVED** [review]

File: _specs_/rest/paths/reference/vXTickers.yml [hunk 1]
```diff
@@ -18,6 +18,17 @@ parameters:
...
       type: string
+  - name: market
+    in: query
+    description: |
+      Specify a market type.
+    required: false
+    schema:
+      type: string
+      enum:
+        - stocks
```
Comment (timetraveler328) [inline _specs_/rest/paths/reference/vXTickers.yml:29]: I made a FE ticket to address adding an empty dropdown item for non-required dropdowns

**Review by timetraveler328 — APPROVED** [review]

File: _specs_/rest/schemas/reference/vXTickers.yml [hunk 2]
```diff
@@ -1,46 +1,54 @@
...
+        description: Whether or not the asset is actively traded. False means the asset has been delisted.
+      currency_name:
+        type: string
+        description: The name of the currency that this asset is traded with.
+      cik:
+        type: string
+        description: The CIK number for this ticker. Find more information [here](https://en.wikipedia.org/wiki/Central_Index_Key).
+      composite_figi:
+        type: string
+        description: The composite OpenFIGI number for this ticker. Find more information [here](https://www.openfigi.com/)
```
Comment (qrpike) [inline _specs_/rest/schemas/reference/vXTickers.yml:44]:
```suggestion
        description: The composite OpenFIGI number for this ticker. Find more information [here](https://www.openfigi.com/assets/content/Open_Symbology_Fields-2a61f8aa4d.pdf)
```

File: _specs_/rest/schemas/reference/vXTickers.yml [hunk 3]
```diff
@@ -1,46 +1,54 @@
...
+      share_class_figi:
+        type: string
+        description: The share Class OpenFIGI number for this ticker. Find more information [here](https://www.openfigi.com/)
```
Comment (qrpike) [inline _specs_/rest/schemas/reference/vXTickers.yml:47]:
```suggestion
        description: The share Class OpenFIGI number for this ticker. Find more information [here](https://www.openfigi.com/assets/content/Open_Symbology_Fields-2a61f8aa4d.pdf)
```

File: _specs_/rest/schemas/reference/vXTickers.yml [hunk 4]
```diff
@@ -1,46 +1,54 @@
...
+  results:
+    type: array
+    description: |
+      An array of tickers that match your query.
+
+      Note: Although you can query by CUSIP, for legal reasons we are not allowed to return the CUSIP
+      in the response body.
+    properties:
+      ticker:
+        $ref: '../index.yml#/StockSymbol'
```
[the remark this answers is not in the document]
Comment (jbonzo) [reply _specs_/rest/schemas/reference/vXTickers.yml:13]: idk this is just the name of the internal swagger spec stuff. Its pretty broken out and make a new field requires making a new file and a new entry in two other files and... its fine right?

**Pull request merged**